From f224d4004c360835720a361a518b8f61f046eae4 Mon Sep 17 00:00:00 2001
From: Gaetan <g.depin@sonoscanner.fr>
Date: Thu, 17 Sep 2026 11:02:28 +0200
Subject: [PATCH] Skip -isystem check

It injects host path:
| ERROR: libreoffice-7.3.0.3-r0 do_configure: QA Issue: This autoconf log indicates errors, it looked at host include and/or library paths while determining system capabilities.
| Rerun configure task after fixing this. [configure-unsafe]
---
 configure.ac | 20 +-------------------
 1 file changed, 1 insertion(+), 19 deletions(-)

diff --git a/configure.ac b/configure.ac
index a9cbbd025..1d2b8a97b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -4376,25 +4376,7 @@ else
 fi
 AC_SUBST(ENABLE_EMSCRIPTEN_PROXY_POSIX_SOCKETS)
 
-# Use -isystem (gcc) if possible, to avoid warnings in 3rd party headers.
-# NOTE: must _not_ be used for bundled external libraries!
-ISYSTEM=
-if test "$GCC" = "yes"; then
-    AC_MSG_CHECKING( for -isystem )
-    save_CFLAGS=$CFLAGS
-    CFLAGS="$CFLAGS -isystem /usr/include -Werror"
-    AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[]], [[ return 0; ]])],[ ISYSTEM="-isystem " ],[])
-    CFLAGS=$save_CFLAGS
-    if test -n "$ISYSTEM"; then
-        AC_MSG_RESULT(yes)
-    else
-        AC_MSG_RESULT(no)
-    fi
-fi
-if test -z "$ISYSTEM"; then
-    # fall back to using -I
-    ISYSTEM=-I
-fi
+ISYSTEM=-isystem
 AC_SUBST(ISYSTEM)
 
 dnl ===================================================================
-- 
2.53.0

